pages/common/cp.md

```diff
@@ -19,6 +19,10 @@
 
 `cp -vR {{path/to/source_directory}} {{path/to/target_directory}}`
 
+- Copy multiple files at once to a directory:
```

Comment:
Thanks for your contribution. The addition is fine for the other page in Linux. But in common isn't this the same as last command?